src/script/start_rabbitmq.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "$BASH_SOURCE[0]" )" &> /dev/null && pwd )
+RABBITMQ_VERSION="rabbitmq:3.13.2-management"
```

Comment:
```suggestion
RABBITMQ_VERSION="rabbitmq:management"
```

Do we mind a particular version? For system test purposes we want to check we continue to work with all new versions

Reply:
shall I change this too?